lib/msal-core/src/Utils.ts

```diff
@@ -140,10 +140,10 @@ export class Utils {
    */
   static expiresIn(expires: string): number {
```

Comment:
Could this be `parseExpiresIn`, as the name doesn't make it clear what is happening. And I think the variable could be `expiresIn` to be even more clear.

Reply:
Changed the names